### ISSUE TITLE
Problem with Jei items

### ISSUE THREAD

**Issue by Dnksh** [iChun/Clef#17]
I have problem with items from mo'creatures and other mods whenever clef is installed.  some items don't have textures, some are just pink-black or just pink or some are just invisible. Is there any fix for this? I love this mod and I wouldn't want to uninstall it.

**Comment by Dnksh** [comment]
![bug](https://user-images.githubusercontent.com/42710750/74101713-1d024e00-4b3d-11ea-959d-b15accf80dca.png)
Here's screen of the problem.

**Comment by iChun** [comment]
Does this happen with just JEI and Clef?

**Comment by Dnksh** [comment]
Yes. Whenever i have it installed some items lose texture and are same as on screen or pink-black. When I uninstall clef items get their textures back

**Comment by Dnksh** [comment]
Oh and whenever I uninstall animania mod some items get their textures and some don't

**Comment by Brycey92** [comment]
I'm also having this rendering issue.

**Comment by Dnksh** [comment]
> I'm also having this rendering issue.

And yet no repply

**Comment by TheFurryDevil-zz** [comment]
A temporary fix is to hide all Clef items in JEI.
To do that activate the "Hide Ingredients Mode" in the JEI config, click on the instruments and perhaps the one block from Clef so they are marked red, and then disable "Hide Ingredients Mode" again.
